src/raygun.rum/vendor/web-vitals.vendor.js

```diff
@@ -1,567 +1,303 @@
 /**
- * This comes from the google web-vital repository, base only script @ https://github.com/GoogleChrome/web-vitals
+ * web-vitals v2.1.0
+ * This comes from the google web-vital repository, base only script @https://github.com/GoogleChrome/web-vitals
  */
-(function () {
-    // This ensures that we do not initilize Core Web Vitals for non-browser environments
-    if(typeof document === 'undefined') {
-      return;
-    }
-
-    /*
-     * Copyright 2020 Google LLC
-     *
-     * Licensed under the Apache License, Version 2.0 (the "License");
-     * you may not use this file except in compliance with the License.
-     * You may obtain a copy of the License at
-     *
-     *     https://www.apache.org/licenses/LICENSE-2.0
-     *
-     * Unless required by applicable law or agreed to in writing, software
-     * distributed under the License is distributed on an "AS IS" BASIS,
-     * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-     * See the License for the specific language governing permissions and
-     * limitations under the License.
-     */
-
-    /**
-     * Performantly generate a unique, 30-char string by combining a version
-     * number, the current timestamp with a 13-digit number integer.
-     * @return {string}
-     */
-    var generateUniqueID = function generateUniqueID() {
-      return "v1-".concat(Date.now(), "-").concat(Math.floor(Math.random() * (9e12 - 1)) + 1e12);
-    };
 
-    /*
-     * Copyright 2020 Google LLC
-     *
-     * Licensed under the Apache License, Version 2.0 (the "License");
-     * you may not use this file except in compliance with the License.
-     * You may obtain a copy of the License at
-     *
-     *     https://www.apache.org/licenses/LICENSE-2.0
-     *
-     * Unless required by applicable law or agreed to in writing, software
-     * distributed under the License is distributed on an "AS IS" BASIS,
-     * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-     * See the License for the specific language governing permissions and
-     * limitations under the License.
-     */
-    var initMetric = function initMetric(name, value) {
-      return {
-        name: name,
-        value: typeof value === 'undefined' ? -1 : value,
-        delta: 0,
-        entries: [],
-        id: generateUniqueID()
-      };
+(function(exports) {
+  'use strict';
+
+  // This ensures that we do not initialize Core Web Vitals for non-browser environments
```

Comment:
This is is an update we added to this script? Do we want to make it more clear that we added this block by tagging things? Not a necessity.

E.g. "Raygun: This  ensures that we ..."